tests/Propel/Tests/Runtime/ActiveQuery/CriteriaTest.php

```diff
@@ -1251,9 +1251,9 @@ public function testGroupBy()
 
         $result = $c->createSelectSql($params);
 
-        if ($this->runningOnPostgreSQL()){
-            $sql = 'SELECT book.id, book.title, book.isbn, book.price, book.publisher_id, book.author_id, COUNT(review.id) AS Count FROM book LEFT JOIN review ON (book.id=review.book_id) GROUP BY book.id,book.title,book.isbn,book.price,book.publisher_id,book.author_id';
-        } else {
+        $sql = $this->getSql('SELECT book.id, book.title, book.isbn, book.price, book.publisher_id, book.author_id, COUNT(review.id) AS Count FROM book LEFT JOIN review ON (book.id=review.book_id) GROUP BY book.id,book.title,book.isbn,book.price,book.publisher_id,book.author_id');
+
+        if ($this->runningOnSQLite()){
```

Comment:
It's not obvious to me why this should change from Postgres to SQLite. Can I have some more information?

Reply:
MySql requires now to have all select columns in the group by part of the query and is more aligned now with Postgres. SQLite hasn't changed.